chemprop/cli/hpopt.py

```diff
@@ -315,11 +315,17 @@ def train_model(config, args, train_dset, val_dset, logger, output_transform, in
     model = build_model(args, train_loader.dataset, output_transform, input_transforms)
     logger.info(model)
 
-    monitor_mode = "min" if model.metrics[0].minimize else "max"
-    logger.debug(f"Evaluation metric: '{model.metrics[0].alias}', mode: '{monitor_mode}'")
+    if args.tracking_metric == "val_loss":
+        tracking_metric_class = model.criterion
+    else:
+        tracking_metric_class = MetricRegistry[args.tracking_metric]
+        args.tracking_metric = "val/" + args.tracking_metric
+
+    monitor_mode = "min" if tracking_metric_class.minimize else "max"
+    logger.debug(f"Evaluation metric: '{tracking_metric_class.alias}', mode: '{monitor_mode}'")
```

Comment:
I see now that `model.criterion` is the `tracking_metric`, but that `MetricRegistry[args.tracking_metric]` is the `tracking_metric_class`. I have changed the first one to also give the class for consistency. I don't need to use `args.tracking_metric.alias` because `args.tracking_metric` is already the string alias for the metric, which is why we can do `MetricRegistry[args.tracking_metric]`.